data/json/items/classes/magazine.json

```diff
@@ -12,6 +12,7 @@
     "color": "ltgray",
     "rigid": false,
     "reliability": 6,
+    "capacity": 9999,
```

Comment:
How much capacity is realistic then? IRL you could make the belt as long as you want if you don't plan to move the gun around.